extensions/apps/antenna/src/components/app-routes/index.tsx

```diff
@@ -39,6 +39,9 @@ import {
 } from './data-loaders';
 import { mapReflectEntryData } from '@akashaorg/ui-awf-hooks';
 import { NotFoundComponent } from './not-found-component';
+import { EditorActions } from '@akashaorg/design-system-components/lib/components/Editor';
+
+const editorActionsRef = createRef<EditorActions>();
```

Comment:
I don't think we need this ref here as it's not used.

Reply:
it is used, it's passed to the reflect editor

Reply:
I meant the actions associated with the ref :)

Reply:
they are used in the reflect editor, to check if editor is empty or not through the children that are exposed on that ref